types/chayns/index.d.ts

```diff
@@ -0,0 +1,938 @@
+// Type definitions for chayns 0.14
+// Project: https://gitlab.tobit.com/chayns-api/chayns-typings
```

Comment:
That is our internal link for the typings project. You are right, the typings are for that project.